src/main.cpp

```diff
@@ -107,13 +108,37 @@ static void deleteInputDevices(QMap<SDL_JoystickID, InputDevice*> *joysticks)
     joysticks->clear();
 }
 
+/**
+ * @brief Function used for copying settings used by previous revisions of antimicrox to provide backward compatibility
+ * TODO remove it later
+ */
+void importLegacySettingsIfExist()
+{
+    qDebug() << "Importing settings";
+    QFileInfo current(PadderCommon::configFilePath());
+    QFileInfo legacy(PadderCommon::configLegacyFilePath());
+    if(legacy.exists() && legacy.isFile())
+    {
+        qDebug() << "Legacy settings found";
+        if(!current.exists())
+        {
+            if(QFile::copy(PadderCommon::configLegacyFilePath(),PadderCommon::configFilePath()))
+            {
+                qDebug() << "Legacy antimicroX settings copied";
```

Comment:
What about deleting the old settings? It's left there as trash and the user doesn't even know about it. We should inform them to manually delete, or we can delete the old settings, risking enraged users if something went wrong. :) (this `QFile::copy` looks pretty safe though)

Reply:
I decided to leave old settings as they were, I don't want to delete anything useful (in old antimicroX directory may be still some old files).

Reply:
Currently, there is mess in code formatting, there are used both versions with space and without. https://github.com/AntiMicroX/antimicroX/issues/25

Reply:
Then just adding a popup that the old directory is safe to delete? Like for the error case.

Reply:
But is it safe for delete? In my case I have found file antimicro.ini (from the first version) of app, deleting it didn't change anything, but just in case I would avoid deleting it.